index.html

```diff
@@ -67,11 +66,167 @@ <h3>
         </div>
         <img src="images/Action.svg" alt="action" />
         <div class="my-recent-works">
-          <h3>My Recent</h3>
-          <h3>Works</h3>
+          <h3>My Recent Works</h3>
           <img src="images/Indicator.png" alt="Indicator">
```

Comment:
- [ ] Well done on styling the heading "Recent works" but kindly move it after the first section because it is overwriting it. KEEP ROCKING 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159126481-e292b371-d318-41de-b7c9-eaf23d9346ad.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/159126498-8e6e5913-957e-4574-b6fa-59eb9703e646.png)